packages/complex-matcher/src/index.spec.js

```diff
@@ -11,11 +11,14 @@ import {
   setPropertyClause,
 } from './'
 
-it('getPropertyClausesStrings', () => {
-  const tmp = getPropertyClausesStrings(parse('foo bar:baz baz:|(foo bar)'))
+it('getPropertyClausesStringsOrRegex', () => {
+  const tmp = getPropertyClausesStringsOrRegex(
+    parse('foo bar:baz baz:|(foo bar) foo:/^bar$/')
```

Comment:
You also need to test it's correctly handled in a `or` because it's a different code path.